packages/dd-trace/src/config.js

```diff
@@ -707,6 +709,7 @@ class Config {
     this._setBoolean(env, 'dsmEnabled', DD_DATA_STREAMS_ENABLED)
     this._setBoolean(env, 'dynamicInstrumentationEnabled', DD_DYNAMIC_INSTRUMENTATION_ENABLED)
     this._setString(env, 'env', DD_ENV || tags.env)
+    this._setBoolean(env, 'traceEnabled', DD_TRACE_ENABLED)
```

Comment:
this will never ever be false because we stop any initialization of the library if DD_TRACE_ENABLED is false here: https://github.com/DataDog/dd-trace-js/blob/bba5f3ddb3e6d693aa74584f8c73d5b5cbf4ea95/packages/dd-trace/src/index.js#L9

Reply:
I included this primarily to enable its transmission in the telemetry payload, if though what will be set will always be true.